src/Game.js

```diff
@@ -87,8 +94,19 @@ export default class Game {
         });
     }
 
-    initColorQueue(width, height) {
+    initQueueContainer(width, height) {
         this.updateCanvas('color-queue', width, height);
+        const text = new createjs.Text("Next", "32px Roboto", "#000000");
+        text.x = 70;
+        text.y = 15;
+        this.queueContainer.addChild(text);
+    }
+
+    updateQueueContainer() {
+        const first = QueueDisplay.createSquare(ColorUtil.rgbaToCSSRgba(this.nextColor), 60, 60);
+        const second = QueueDisplay.createSquare(this.colorQueue.getQueue()[0], 60, 160);
+        const third = QueueDisplay.createSquare(this.colorQueue.getQueue()[1], 60, 260);
+        this.queueContainer.addChild(first, second, third);
```

Comment:
This will continuously add new square objects to the stage on every move event, leaving previous squares behind (also literally 'behind' the other squares). Recommend creating squares only once then updating their references graphical components much like ColorShape. Else if you are looking to animate or continue instantiating new squares, please garbage collect squares you no longer use.